pkg/scheduler/actions/backfill/backfill.go

```diff
@@ -68,6 +74,101 @@ func (alloc *backfillAction) Execute(ssn *framework.Session) {
 			}
 		}
 	}
+
+	if len(alloc.arguments) == 0 {
+		return
+	}
+
+	backfillNonBestEffortJobsEnabled, err := strconv.ParseBool(alloc.arguments[conf.BackfillFlagName])
```

Comment:
refer to `GetInt` in Arguments for that.

Reply:
fixed